llarp/router/router.cpp

```diff
@@ -97,6 +97,84 @@ namespace llarp
     }
   }
 
+  util::StatusObject
+  Router::ExtractSummaryStatus() const
+  {
+    if (_running)
+    {
+      auto services = _hiddenServiceContext.ExtractStatus();
+      auto link_types = _linkManager.ExtractStatus();
+
+      int tx_rate = 0;
+      int rx_rate = 0;
+      int peers = 0;
+      for (const auto& links : link_types)
+      {
+        if (links == nlohmann::json::array())
```

Comment:
I think `if (links.empty())` would work here (and 4 lines down).

Reply:
Actually, this check doesn't look needed at all: if it's empty then the next for loop won't do anything anyway.